terraform-provider-constellation/internal/provider/provider.go

```diff
@@ -71,7 +71,7 @@ func (p *ConstellationProvider) Configure(ctx context.Context, req provider.Conf
 	config := datastruct.ProviderData{}
 
 	// Make the clients available during data source and resource "Configure" methods.
-	resp.DataSourceData = config
+	// resp.DataSourceData = config // TODO check with moritz
```

Comment:
I think it doesn't have any disadvantage to have this set all the time, does it? Don't have any real opinion on this though